tests/base-qemu/base.yaml

```diff
@@ -1,4 +1,4 @@
-version: 1
+$schema: https://toit.io/schemas/artemis/pod-specification/v1.json
```

Comment:
Did we already update this? I don't think so. We should push the new one out (the one that doesn't have chip support).

Reply:
it still has the "chip". We need to update it.